libs/api/Verifier.cpp

```diff
@@ -330,9 +330,7 @@ load_byte_code(
 
             program->handle = ebpf_handle_invalid;
             program->program_type = *(const GUID*)raw_program.info.type.platform_specific_data;
-
-            // Only autoload programs with a defined program type.
-            program->autoload = memcmp(&program->program_type, &empty_program_type, sizeof(empty_program_type)) != 0;
+            program->autoload = true;
```

Comment:
nit: question: why did we need to change this?

Reply:
1. The link_tests test was failing.
2. Libbpf always sets autoload = true.